test/org/opensolaris/opengrok/configuration/RuntimeEnvironmentTest.java

```diff
@@ -510,9 +510,9 @@ public void testCanAcceptMessage() throws Exception {
         Assert.assertFalse(instance.canAcceptMessage(m1));
         m1.setExpiration(new Date(System.currentTimeMillis() - 1));
         Assert.assertFalse(instance.canAcceptMessage(m1));
-        m1.setExpiration(new Date(System.currentTimeMillis() - 0));
+        m1.setExpiration(new Date(System.currentTimeMillis() + 50));
         Assert.assertTrue(instance.canAcceptMessage(m1));
-        m1.setExpiration(new Date(System.currentTimeMillis() + 1));
+        m1.setExpiration(new Date(System.currentTimeMillis() + 100));
```

Comment:
Appeared as a too little marging for testing `isExpired` when running just few tests (little load).